config/application.rb

```diff
@@ -1,8 +1,20 @@
 # frozen_string_literal: true
 
-require File.expand_path('../boot', __FILE__)
+
+require_relative 'boot'
 
 require 'rails/all'
+require "active_model/railtie"
```

Comment:
Looks fine but why does this need to change?

Reply:
Why go from `rails/all` I mean. I know there can be reasons, just wondering

Reply:
active_storage issue we can solve  in 2 ways

1) Install active storage  and add db migrations, configs

2) Disable active storage.
we do not want to load all  "rails/all".